import-export-cli/cmd/deprecated/exportAPI.go

```diff
@@ -106,6 +112,8 @@ func init() {
 		"Version of the API to be exported")
 	ExportAPICmdDeprecated.Flags().StringVarP(&exportProvider, "provider", "r", "",
 		"Provider of the API")
+	ExportAPICmdDeprecated.Flags().StringVarP(&exportRevisionNum, "rev", "", "",
+		"Revision number of the API to be exported")
```

Comment:
Here as well. Do we need to add this to deprecated commands?

Reply:
removed the implementation from deprecated command. cf58769